build.sbt

```diff
@@ -105,18 +105,23 @@ lazy val validation = (project in file("validation"))
 
 lazy val panel = (project in file("panel"))
   .settings(hmdaBuildSettings: _*)
+  .settings(Revolver.settings:_*)
   .settings(
-    assemblyMergeStrategy in assembly := {
-      case "application.conf" => MergeStrategy.concat
-      case "META-INF/io.netty.versions.properties" => MergeStrategy.concat
-      case x =>
-        val oldStrategy = (assemblyMergeStrategy in assembly).value
-        oldStrategy(x)
-    },
-    libraryDependencies ++= akkaPersistenceDeps ++ httpDeps
+    Seq(
+      assemblyJarName in assembly := {s"${name.value}.jar"},
```

Comment:
No, I ran `sbt clean assembly`, like I said. Maybe add that it has to be packaged separately to the documentation?